src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -336,6 +336,8 @@ void initialize(RTPLevelRelayType rtpLevelRelayType)
     {
         super.initialize(rtpLevelRelayType);
 
+        bitrateController.update(null, -1);
```

Comment:
so this obsolesces https://github.com/jitsi/jitsi-videobridge/pull/527, ya?

Reply:
right, just closed it.